### ISSUE TITLE
norm image or not?

### ISSUE THREAD

**Issue by ahangchen** [svip-lab/FastMVSNet#9]
In train set, norm_image is commented:https://github.com/svip-lab/FastMVSNet/blob/master/fastmvsnet/dataset.py#L111
while in test set, norm_image shows up:https://github.com/svip-lab/FastMVSNet/blob/master/fastmvsnet/dataset.py#L289

Whether we need to norm images to reproduce the result in the paper? Which mode is used when training the pretrained.pth in outputs directory?

**Comment by niujinshuchong** [comment]
@ahangchen 

Thanks for pointing this out. I think it is a bug. It's better to make it consistent in training and testing.

The provided pretrained.pth is trained without norm_image. 

I just test the model without norm_image in dtu datasets. The result is quite similar.

|  |      Acc.      |  Comp. | Overall |
|----------|:-------------:|------:|------:|
|with norm_image | 0.3405 |  0.4119  |  0.3762|
|without norm_image |  0.3410|  0.4099   | 0.3755 |

Batch norm is used in testing, so after first convolution layer, the feature map is normalized. I think this is the reason why two results are similar.

**Comment by ahangchen** [comment]
Got it, thank you very much.
